packages/charts/src/renderers/xy-renderer.ts

```diff
@@ -18,5 +29,17 @@ export abstract class XYRenderer<TA extends IAccessors> extends Renderer<TA> {
             y: scales.y.convert(dataSeries.accessors.data.y(point, index, dataSeries.data, dataSeries)),
         };
     }
+    public getDataPointIndex(series: IDataSeries<TA>, values: { [p: string]: any }, scales: Scales): number {
```

Comment:
newline would be good
```suggestion

    public getDataPointIndex(series: IDataSeries<TA>, values: { [p: string]: any }, scales: Scales): number {
```

Reply:
It looks like AreaRenderer has an identical implementation of this method, so I think AreaRenderer's override of getDataPointIndex can be removed.

Reply:
We should transfer this comment from LineRenderer;

```suggestion
    /** See {@link Renderer#getDataPointIndex} */
    public getDataPointIndex(series: IDataSeries<TA>, values: { [p: string]: any }, scales: Scales): number {
```